### PR TITLE
Adding CLMS HRSI visualisations

### PR DESCRIPTION
**Pull request by willrayeo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -582,7 +582,30 @@ Vegetation Phenology and Productivity Parameters is available in Sentinel Hub, r
 - [RSLOPE visualisation script](copernicus_services/vpp-slope-of-senescent-period-rslope)
 - [LSLOPE visualisation script](copernicus_services/vpp-slope-of-greening-up-period-lslope)
 - [SPROD visualisation script](copernicus_services/vpp-seasonal-productivity-sprod)
-- [TPROD visualisation script](copernicus_services/vpp-total-productivity-tprod) 
+- [TPROD visualisation script](copernicus_services/vpp-total-productivity-tprod)
+
+### High Resolution Snow & Ice Monitoring
+
+The High-Resolution Snow & Ice Monitoring service ([HR-S&I](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring)) is part of the Copernicus Land Monitoring Service (CLMS). The snow aspect of the service provides products measuring Snow cover (FSC, FSTOC, FSCOG, GFSC), Snow state conditions (WDS, SWS) and persistent snow area (PSA). There are aso ice products that measure ice cover (RLIE) and aggregated river and lake ice extent (ARLIE).
```
Comment (Dorothyrono) [inline README.md:589]: For a chance to describe each collection, I suggest to have an entry for each collection/product separately and list the associated scripts underneath.

Comment (willrayeo) [reply README.md:589]: Sure, will split them into the seperate collections so there isn't one very long list!

File: copernicus_services/hrsi-fsc-ndsi/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+## Description of representative images
+*23rd April 2020, Innsbruck, AT* 
+
+![NDSI](fig/figure.png)  
+
+## Resources
+
+- [Data source](https://land.copernicus.eu/pan-european/biophysical-parameters/high-resolution-snow-and-ice-monitoring/snow-products)
+
+- Entry in public collections
```
Comment (jonasViehweger) [inline copernicus_services/hrsi-fsc-ndsi/README.md:24]: I would expect a link here, is that going to be added at a later time? If so I think it is better to add it when the public collections entry is available and not have some text pointing to nothing.

File: README.md [hunk 3]
```diff
@@ -582,13 +582,65 @@ Vegetation Phenology and Productivity Parameters is available in Sentinel Hub, r
...
+#### Fractional Snow Cover
+
+The Fractional Snow Cover (FSC) product provides the snow fraction at the Top Of Canopy (FSCTOC) and On Ground (FSCOG). NDSI is also provided as part of this service.
+
+##### Available scripts
+
+- [FSC FSCOG visualisation script](copernicus_services/hrsi-fsc-on-ground-fsi)
+- [FSC FSCOG quality visualisation script](copernicus_services/hrsi-fsc-on-ground-fsi-quality)
+- [FSC FSTOC visualisation script](copernicus_services/hrsi-fsc-top-of-canopy)
+- [FSC FSTOC quality visualisation script](copernicus_services/hrsi-fsc-top-of-canopy-quality)
```
Comment (jonasViehweger) [inline README.md:600]: This link doesn't work

File: README.md [hunk 4]
```diff
@@ -582,13 +582,65 @@ Vegetation Phenology and Productivity Parameters is available in Sentinel Hub, r
...
+
+#### Fractional Snow Cover
+
+The Fractional Snow Cover (FSC) product provides the snow fraction at the Top Of Canopy (FSCTOC) and On Ground (FSCOG). NDSI is also provided as part of this service.
+
+##### Available scripts
+
+- [FSC FSCOG visualisation script](copernicus_services/hrsi-fsc-on-ground-fsi)
+- [FSC FSCOG quality visualisation script](copernicus_services/hrsi-fsc-on-ground-fsi-quality)
+- [FSC FSTOC visualisation script](copernicus_services/hrsi-fsc-top-of-canopy)
```
Comment (jonasViehweger) [inline README.md:599]: Same with this

**Review by jonasViehweger — CHANGES_REQUESTED** [review]

**Pull request merged**